src/GitHub.Logging/Logging/ILoggerExtensions.cs

```diff
@@ -25,5 +27,35 @@ public static void Assert(this ILogger logger, bool condition, string messageTem
 #pragma warning restore Serilog004
             }
         }
+
+        public static void Time(this ILogger logger, string name, Action method)
+        {
+            var startTime = DateTime.Now;
+            method();
+            logger.Information("{Name} took {Seconds:0.00} seconds", name, (DateTime.Now - startTime).TotalSeconds);
```

Comment:
@StanleyGoldman any idea what `{Seconds:0.00}` isn't formatting the `TotalSeconds` double? 😕

Reply:
Since Serilog is a structured logger, you aren't looking at a traditional format string here.

```suggestion
            logger.Information("{Name} took {Seconds} seconds", name, (DateTime.Now - startTime).TotalSeconds.ToString("0.##"));
```